network/dns_frame.ksy

```diff
@@ -0,0 +1,121 @@
+meta:
+  id: dns_packet
```

Comment:
File should be named "dns_packet.ksy" as well. As far as I see, RFC never refers to DNS packet as "DNS frame", as file name suggests.